Philip_Cron/assignments/wizard_ninja_samurai/wizard_spec.rb

```diff
@@ -0,0 +1,22 @@
+require_relative 'wizard'
+require_relative 'ninja'
+RSpec.describe Wizard do
+    before do
+        @wiz = Wizard.new
+    end
+    it 'Wizard has a default health of 50 and intelligence of 25' do
+        expect(@wiz.health).to eq(50)
+    end
+    it 'Wizard has a heal method that increases health by 10' do
+        expect(@wiz.heal).to eq(60)
+    end
+    it 'Wizard has an ancestor chain that includes Human' do
+        yes = true
+        expect(@wiz.class.ancestors.include?(Human)).to eq(yes)
```

Comment:
you shouldn't need to create a variable here to equal true.  The .include returns either true or false so you could just  put .to eq(true)